### DISCUSSION TITLE
Multi Token

### DISCUSSION THREAD

**Discussion by jorensc** [Vedza/NitroSniperGo#59] [Ideas]
Multiple Alt tokens and 1 token to redeem on

**Comment by Vedza** [comment]
I've never added this feature cause first I didn't need it to get Nitro but also because this might make your account banned because of too many requests made.

**Reply by Vedza** [reply]
Damn you're lucky, I made a version with alts for me and I got banned twice a few months ago.

**Comment by Vedza** [comment]
I've added this feature on branch `v2`, need to do some tests before release but it seems it works.
